andela/ah-rambo-frontend
# #167190490 Implement Forgot Password Page

#### What does this PR do?
- Implement forgot password page
- Add unit tests

#### Description of Task proposed in this pull request? 
- Add request for password components, action types, actions and reducers
- Style the forgot password pages
- Consume endpoints to implement the functionality
- Write components unit test 

#### How should this be manually tested (Quality Assurance)?
- Run `npm run start:dev'
- A browser tab will launch, then navigate to this route ```/request-password```
- Input a valid email address and confirm the email address is a registered one, then ```submit```.
- If successful, check your mail to get the link.

#### What are the relevant pivotal tracker stories?
- [#167190490](https://www.pivotaltracker.com/story/show/167190490)

#### Any background context you want to add (Operations Impact)?
- Run ```npm install``` to install all necessary dependencies. 

#### What I have learned working on this feature: 
- I understood how redux works and state management

#### Screenshots: 
<img width="1440" alt="Screenshot 2019-09-08 at 2 46 04 PM" src="https://user-images.githubusercontent.com/42908597/64489153-7c36fc00-d247-11e9-9125-3bab1f873952.png">
